src/lib/isMACAddress.js

```diff
@@ -2,11 +2,13 @@ import assertString from './util/assertString';
 
 const macAddress = /^([0-9a-fA-F][0-9a-fA-F]:){5}([0-9a-fA-F][0-9a-fA-F])$/;
 const macAddressNoColons = /^([0-9a-fA-F]){12}$/;
+const macAddressUsingHyphen = /^([0-9a-fA-F][0-9a-fA-F]-){5}([0-9a-fA-F][0-9a-fA-F])$/;
+const macAddressUsingSpaces = /^([0-9a-fA-F][0-9a-fA-F]\s){5}([0-9a-fA-F][0-9a-fA-F])$/;
 
 export default function isMACAddress(str, options) {
   assertString(str);
   if (options && options.no_colons) {
```

Comment:
I was also still figuring out, why do we need an option for `colons`? 
Instead of just allowing different cases of validations. 

Cc. @profnandaa @chriso

Reply:
Sure, that might not be necessary.

Reply:
To make sure that I don't break on things, I will leave it for now. 
And propose changes. 
Thanks, @profnandaa.